readme.md

```diff
@@ -372,17 +344,17 @@ It creates getter and setter methods for the `name` instance variable.
 
 ```rb
 class User
-  attr_reader :firstname
-  attr_writer :lastname
+attr_reader :firstname
+attr_writer :lastname
 
-  def initialize(firstname, lastname)
-    @firstname = firstname
-    @lastname = lastname
-  end
+def initialize(firstname, lastname)
+@firstname = firstname
+@lastname = lastname
+end
 
-  def full_name
-    return "#{@firstname.capitalize} #{@lastname.capitalize}"
-  end
+def full_name
+return "#{@firstname.capitalize} #{@lastname.capitalize}"
+end
 
 end
```

Comment:
might be valuable to mention why you would want only a read method or only a set method.